src/client/apps/edit/components/admin/components/appearances.tsx

```diff
@@ -1,50 +1,50 @@
+import { Box, Flex } from "@artsy/palette"
 import React from "react"
-import { Col, Row } from "react-styled-flexboxgrid"
-import AutocompleteListMetaphysics from "client/components/autocomplete2/list_metaphysics"
+import AutocompleteListMetaphysics from "../../../../../components/autocomplete2/list_metaphysics"
 
-export const AdminAppearances = props => {
+export const AdminAppearances = () => {
   return (
-    <div>
-      <Row>
-        <Col xs={6}>
+    <React.Fragment>
+      <Flex flexDirection={["column", "row"]}>
+        <Box width={["100%", "50%"]} pr={[0, 20]}>
```

Comment:
Should be using [design-system `space` values](https://github.com/artsy/palette/blob/master/src/Theme.tsx#L125). 20px -> 2. This applies to all following code; 40 -> 4, etc. If a value is missing, it's best to find the next closest item so that our designs can remain consistent.

Reply:
Maybe we don't have the theme provider here, these are not resolving correctly for me.

Reply:
Ah yeah, should add 
```js
import { Theme } from  '@artsy/palette'
```

and put it at the root of your react tree.